pkg/master/master.go

```diff
@@ -544,7 +554,34 @@ func (m *Master) init(c *Config) {
 	}
 
 	// TODO: Attempt clean shutdown?
-	m.masterServices.Start()
+	if m.enableCoreControllers {
+		m.NewBootstrapController().Start()
+	}
+}
+
+// NewBootstrapController returns a controller for watching the core capabilities of the master.
+func (m *Master) NewBootstrapController() *Controller {
```

Comment:
I see.  I am sympathetic, but I want to be really careful about leaving debris around the kube codebase to accomodate out-of-tree stuff.  This is the sort of thing that is easily broken when some static analysis tool says "hey this could be private" and then we change it.